pkg/quota/openstack/limits.go

```diff
@@ -0,0 +1,92 @@
+package openstack
```

Comment:
I would even go further and move all of this into `pkg/asset/installconfig/openstack/validation/cloudinfo.go`.

Reply:
ack, I'm looking at it now.